custom_components/ha-glocaltokens/entity.py

```diff
@@ -12,24 +13,11 @@ def __init__(self, coordinator, config_entry):
         super().__init__(coordinator)
         self.config_entry = config_entry
 
-    @property
-    def unique_id(self):
-        """Return a unique ID to use for this entity."""
-        return self._name
-
     @property
     def device_info(self):
         return {
-            "identifiers": {(DOMAIN, self.unique_id)},
-            "name": NAME,
```

Comment:
I am afraid we might need to keep `unique_id` property method untouched. It comes from the integration blueprint and might be used internally in HA, so even though it looks redundant, i would suggest we keep it